src/views/Visualizer/View/View.js

```diff
@@ -208,11 +209,16 @@ export class Visualizer extends React.Component {
     if (interactions !== prevInteractions) {
       this.setInteractions();
     }
+
+    if (layersTree !== prevLayersTree) {
+      // noop
+    }
```

Comment:
I don't get the purpose of this condition?

Reply:
Not the slightest now!